opennlp-tools/src/test/java/opennlp/tools/util/wordvector/GloveTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package opennlp.tools.util.wordvector;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+/*
+ * Note:
+ * Examples taken from the 'glove.6B.50d.txt' data set,
+ * which is licensed under CC0 1.0 Universal (CC0 1.0) Public Domain Dedication.
+ * (see: https://creativecommons.org/publicdomain/zero/1.0/)
```

Comment:
:+1: do you think that should go in a `NOTICE` file somewhere too?

Reply:
For that case (CC0), I'm unsure if this is required, given that it is "Public Domain". Maybe, @jzonthemtn knows whether that should be done, or isn't necessary. 

Other opinions/comments also welcome, oc.

Reply:
Just had a read at CC0, and it doesn't require any attribution, so I think we don't need the entry in NOTICE (as per [this asf doc](https://www.apache.org/licenses/LICENSE-2.0.html#redistribution)). But we can leave it open for others to comment when they review this PR too :+1: Thanks @mawiesne !

Reply:
(also searched /apache using GitHub UI, and found only a few projects doing so, but doesn't mean it's needed - e.g. https://github.com/apache/kafka/blob/52f87e2c65220a76f9c21d7da28004bf074d02b3/NOTICE-binary#L400-L403)

Reply:
Some more information: https://www.apache.org/legal/resolved.html#handling-public-domain-licensed-works
However, I don't think, that it needs an entry in NOTICE.